tomography_preprocessing/tilt_series_alignment/aretomo.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from typing import Optional
+
+import typer
+from lil_aretomo.aretomo import run_aretomo_alignment
+
+from ._cli import cli
+from .. import utils
+
+ARETOMO_ALIGNMENT_COMMAND_NAME = 'AreTomo'
+
+@cli.command(name=ARETOMO_ALIGNMENT_COMMAND_NAME)
+def aretomo_function(
+        tilt_series_star_file: Path = typer.Option(...),
+        output_directory: Path = typer.Option(...),
+        exe: Path = typer.Option(...),	
+        local_align: Optional[bool] = typer.Option(False),
+        target_pixel_size: Optional[float] = typer.Option(10),
+        patch_in_x: Optional[float] = typer.Option(5),
+        patch_in_y: Optional[float] = typer.Option(4),
+        correct_tilt_angle_offset: Optional[bool] = typer.Option(False),
+        tomogram_name: Optional[str] = typer.Option(None)
+):
+    tilt_series_metadata = utils.star.iterate_tilt_series_metadata(
+        tilt_series_star_file=tilt_series_star_file,
+        tilt_series_id=tomogram_name
+    )
+    for tilt_series_id, tilt_series_df, tilt_image_df in tilt_series_metadata:
+        utils.aretomo.align_single_tilt_series(
+            tilt_series_id=tilt_series_id,
+            tilt_series_df=tilt_series_df,
+            tilt_image_df=tilt_image_df,
+            alignment_function=run_aretomo_alignment,
+            alignment_function_kwargs={'exe': exe,
+	    'local_align': local_align,
+	    'target_pixel_size': target_pixel_size,
+	    'patch_in_x': patch_in_x,
+	    'patch_in_y': patch_in_y,
+	    'correct_tilt_angle_offset': correct_tilt_angle_offset
+	    },
+            output_directory=output_directory,
+        )
```

Comment:
I see you've mirrored what was done for IMOD (which is a little more complex to reduce code duplication). This looks great but if there will only ever be one AreTomo alignment command it might not be necessary - I leave it up to you :)


i.e.
- you might not need to pass the specific alignment function and its kwargs here, you could just call this directly in `utils.aretomo.align_single_tilt_series`
- you could pass the kwargs as arguments directly to `utils.aretomo.align_single_tilt_series`

Reply:
passed kwargs directly now (silly mistake!) and simplified in commit 'removed kwargs'